### PR TITLE
Refresh 'Contributing' page

### PR DESCRIPTION
**Pull request by erciccione**
More detailed and updated info, updated links, added button for CCS and made several cosmetic changes

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -328,30 +328,42 @@ contributing:
   intro: Monero is an open-source, community-driven project. Described below are several ways to support the project.
   network: Support the Network
   develop: Develop
-  develop_para1: Monero is primarily written in C++. As it is a decentralized project, anyone is welcome to add or make changes to existing code. Pull requests are merged based on community consensus. See the
-  develop_para2: repositories
-  develop_para3: and outstanding
-  develop_para4: issues.
-  full-node: Run a Full Node
-  full-node_para: Run monerod with port 18080 open. Running a full node ensures maximum privacy when transacting with Monero. It also improves distribution of the blockchain to new users.
+  developp: "Monero is open source and permissionless, contributors are welcome and encouraged. A developer may contribute in numerous way to different projects:"
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:331]: permissionless, contributors -> permissionless; contributors

File: _i18n/en.yml [hunk 2]
```diff
@@ -328,30 +328,42 @@ contributing:
...
+  cli: "CLI wallet and Daemon: "
+  cli_p: The core wallet, mostly written in C++.
+  gui: "GUI Wallet: "
+  gui_p: Mostly QML and C++
+  website: "This website: "
+  website_p: Built with Jekyll. No Javascript
+  bug: "Bug and vulnerability hunting: Monero is listed on Hackerone and developers are asked to responsibly disclose bugs and vulnerabilities. See"
+  discl: The Monero Project Vulnerability Response Process
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:339]: add . at end for consistency

File: _i18n/en.yml [hunk 3]
```diff
@@ -328,30 +328,42 @@ contributing:
...
+  develop_descr: >
+    This list only includes the tools stewarded by the core team, but the Monero ecosystem is much more vaste. People can contribute to the development of libraries, services, documentation, graphics, etc. The list is virtually infinite. Contact or join the workgroup more affine to your way of contributing.
+  develop_wg: See the Monero Workgroups
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:342]: add . for consistency

File: _i18n/en.yml [hunk 4]
```diff
@@ -328,30 +328,42 @@ contributing:
...
+  allguides: See all User Guides
   mine: Mine
-  mine_para1: Mining ensures the Monero network remains decentralized and secure. In the Monero graphical user interface and command-line interface, background mining may be activated. Additional resources for mining may be viewed
-  mine_para2: here.
-  ffs: View the Community Crowdfunding System
-  ffs_para1: Monero utilizes a
-  ffs_para2: community crowdfunding system
-  ffs_para3: whereby projects are proposed for development and community-funded. Funding is held in escrow and remunerated to developers once programming milestones are achieved. Anyone may generate new proposals or fund existing ones.
+  mine_p: >
+    Mining ensures the safety of the network. Monero uses @randomx, an asic-resistant algorithm developed by Monero contributors, which aims to abide to the philosophy 1 cpu = 1 vote. For more info about mining see the
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:349]: this seems too Satoshi-cult to me. How about "which aims to remain mineable by common consumer-grade hardware."

Also, why is OpenAlias capitalized correctly but RandomX is not?

File: _i18n/en.yml [hunk 5]
```diff
@@ -328,30 +328,42 @@ contributing:
...
-  develop_para2: repositories
-  develop_para3: and outstanding
-  develop_para4: issues.
-  full-node: Run a Full Node
-  full-node_para: Run monerod with port 18080 open. Running a full node ensures maximum privacy when transacting with Monero. It also improves distribution of the blockchain to new users.
+  developp: "Monero is open source and permissionless, contributors are welcome and encouraged. A developer may contribute in numerous way to different projects:"
+  cli: "CLI wallet and Daemon: "
+  cli_p: The core wallet, mostly written in C++.
+  gui: "GUI Wallet: "
+  gui_p: Mostly QML and C++
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:335]: add . at end

File: _i18n/en.yml [hunk 6]
```diff
@@ -328,30 +328,42 @@ contributing:
...
+  website_p: Built with Jekyll. No Javascript
+  bug: "Bug and vulnerability hunting: Monero is listed on Hackerone and developers are asked to responsibly disclose bugs and vulnerabilities. See"
+  discl: The Monero Project Vulnerability Response Process
+  develop_descr: >
+    This list only includes the tools stewarded by the core team, but the Monero ecosystem is much more vaste. People can contribute to the development of libraries, services, documentation, graphics, etc. The list is virtually infinite. Contact or join the workgroup more affine to your way of contributing.
+  develop_wg: See the Monero Workgroups
+  full-node: Run a Node
+  full-node_p: An easy and effective way to help the Monero network is to run a @node. Nodes ensure the network keeps running safe and decentralized. A simple fully synchronized node is enough to help the network, but if you want to go out of your way, you could run an open @remote-node, to allow other people to connect to it.
+  full-node_p2: "We have several guides to help you run and configure your node:"
+  allguides: See all User Guides
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:346]: add . at end

File: _i18n/en.yml [hunk 7]
```diff
@@ -328,30 +328,42 @@ contributing:
...
-  develop_para4: issues.
-  full-node: Run a Full Node
-  full-node_para: Run monerod with port 18080 open. Running a full node ensures maximum privacy when transacting with Monero. It also improves distribution of the blockchain to new users.
+  developp: "Monero is open source and permissionless, contributors are welcome and encouraged. A developer may contribute in numerous way to different projects:"
+  cli: "CLI wallet and Daemon: "
+  cli_p: The core wallet, mostly written in C++.
+  gui: "GUI Wallet: "
+  gui_p: Mostly QML and C++
+  website: "This website: "
+  website_p: Built with Jekyll. No Javascript
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:337]: add . at end

**Review by SamsungGalaxyPlayer — CHANGES_REQUESTED** [review]

File: _i18n/en.yml (hunk 4 above)
Comment (erciccione) [reply _i18n/en.yml:349]: > this seems too Satoshi-cult to me.

I used a commonly known phrase, no problem for me to change it.

> why is OpenAlias capitalized correctly but RandomX is not?

It doesn't really matter. Those @word are only needed to trigger the moneropedia script, the result willl be standardized anyway (see #1352)

File: _i18n/en.yml [hunk 8]
```diff
@@ -328,30 +328,42 @@ contributing:
...
+  allguides: See all User Guides.
   mine: Mine
-  mine_para1: Mining ensures the Monero network remains decentralized and secure. In the Monero graphical user interface and command-line interface, background mining may be activated. Additional resources for mining may be viewed
-  mine_para2: here.
-  ffs: View the Community Crowdfunding System
-  ffs_para1: Monero utilizes a
-  ffs_para2: community crowdfunding system
-  ffs_para3: whereby projects are proposed for development and community-funded. Funding is held in escrow and remunerated to developers once programming milestones are achieved. Anyone may generate new proposals or fund existing ones.
+  mine_p: >
+    Mining ensures the safety of the network. Monero uses @randomx, an asic-resistant algorithm developed by Monero contributors, which aims to remain mineable by common consumer-grade hardware. For more info about mining see the
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:349]: capitalize ASIC. Sorry, missed this earlier

**Review by SamsungGalaxyPlayer — CHANGES_REQUESTED** [review]

File: _i18n/en.yml (hunk 8 above)
Comment (erciccione) [reply _i18n/en.yml]: Fixed. Thanks.

**Review by SamsungGalaxyPlayer — APPROVED** [review]

**Pull request merged**